build-python-host-emscripten.sh

```diff
@@ -22,16 +22,21 @@ pushd ../usr/local
 # includes aren't need for distribution, various libraries
 # won't be used in the web (at least for now)
 rm -rf bin include lib/pkgconfig lib/libpython3.11d.a
-rm -rf lib/python3.11/test
+rm -rf lib/python3.11/_osx_support.py
 rm -rf lib/python3.11/config-3.11d
+rm -rf lib/python3.11/curses
+rm -rf lib/python3.11/dbm
+rm -rf lib/python3.11/distutils/command/wininst-*.exe
```

Comment:
distutils is not useful in browser context and deprecated anyway, let's remove it completely.

```suggestion
rm -rf lib/python3.11/distutils
```